src/screens/Instructor/Profile/Components/ProfileCard.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Component for Instructor Profile Page
+ * - contents instructor info
+ */
+
+import React from 'react'
+import { Card } from 'react-bootstrap'
+const profileImg = require('../../../../images/icons/profile1.png')
+
+export function ProfileCard(props) {
+  return (
+    <Card className="ip-profile" border="light">
+      <Card.Img className="img" src={profileImg}/>
+      <Card.Text className="name">
+        {props.instructor.name}<br/>
+        <span>University of Illinois at Urbana Champaign</span>
```

Comment:
Why is this hard coded?

Reply:
Oh this is because as I mentioned I cannot get the user's university so I am assuming it is uiuc